packages/loader/src/ShaderLoader.ts

```
@@ -0,0 +1,41 @@
+            return resourceManager.load({ type: "ShaderChunk", url: resource, params: { includeKey: path } });
+          }
+        })
+      ).then(() => {
+        return Shader.create(code);
+      });
+    });
+  }
+
+  private getBuiltinShader(code: string) {
+    const match = code.match(/^\s*\/\/\s*@builtin\s+(\w+)/);
+    if (match && match[1]) return match[1];
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)